apiserver/common/crossmodel/interface.go

```diff
@@ -234,10 +236,10 @@ type Charm interface {
 // RemoteApplication represents the state of an application hosted in an external
 // (remote) model.
 type RemoteApplication interface {
-	// Destroy ensures that this remote application reference and all its relations
-	// will be removed at some point; if no relation involving the
-	// application has any units in scope, they are all removed immediately.
-	Destroy() error
+	// DestroyWithForce in addition to doing what Destroy() does,
+	// when force is passed in as 'true', forces th destruction of remote application,
+	// ignoring errors.
+	DestroyWithForce(bool, time.Duration) (opErrs []error, err error)
```

Comment:
I know the arg names aren't required here, but they do help with readability, for example, the doc comment here says "when force is passed" but then `force` is not a named arg. And presumably the duration is a timeout, but it's not clear.

Reply:
Added arg names